### ISSUE TITLE
:docs: Terrible documentation regarding Multicalls

### ISSUE THREAD

**Issue by AlphaDio** [TrueFiEng/useDApp#770]
Multicalls are a confusing, obnoxious, and very obscure feature.

I spent an hour debugging my application because of an impossibility to deploy a Multicall contract.

I also have trouble with Multicall when it comes to deployments on non-Mainnet chains.

I would ask for a better documentation and troubleshotting of Multicall, and also list of Multicall contracts depending on chains.

Thank you.

**Comment by rzadp** [comment]
Agree, the multicall is a central piece of usedapp and should be covered as much as possible in the docs.

A list of multicall contracts depending on chains is not in the docs, but if you need it right now you can take a look at [this folder](https://github.com/TrueFiEng/useDApp/tree/master/packages/core/src/model/chain) - every chain has a file with a multicall address there.

**Comment by akornato** [comment]
I totally agree with @AlphaDio - I'm trying to use `useDapp` against a local Hardhat network and I was seeing my local app trying to deploy some contract as soon as I connected with the app, and I had no idea why that was happening. I even thought my Metamask was hacked 🤣 Finally I found out it's this multicall autodeploy feature, but does it even make sense when against a local Hardhat network? I guess the contract has to be deployed via Metamask every time I rerun a Hardhat node? And even then, how do I stop it trying to deploy on every reload? If I add any address for `31337` network to `multicallAddresses` in the config, then it won't try to deploy ever again, which is great, but also when I rerun a Hardhat node, i.e. even when there's no multicall contract deployed. This workaround works (I hope nothing breaks) but I think there should be some explicit config option to turn multicall off for some networks?
